doc/articles/platform-specific-csharp.md

```diff
@@ -13,7 +13,7 @@ This guide covers multiple approaches to managing per-platform code in C#. See [
 There are two ways to restrict code or XAML markup to be used only on a specific platform:
 
 * Use [conditionals](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if) within a shared file
-* Place the code in a file which is only included in the desired platform head.
+* Place the code in a separate file which is only included in the desired platform.
```

Comment:
The two approaches are almost the same. Both uses `#if` for files in the class library project.

But LGTM given the correctness of the change.